src/App.js

```diff
@@ -1,16 +1,43 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./Clock.js";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <Container className="grid">
+            <Row>
```

Comment:
you could use a `.map` to handle repeated components. you can take a look at today's pre-class material for some references on how to do it: https://bc.rocketacademy.co/1-frontend/1.3-react

try reading some of [this discussion](https://github.com/rocketacademy/world-clock-bootcamp/pull/8/files#r974245222) to understand how you can turn repeated code into their own components